site/_docs/continuous-integration.md

```diff
@@ -118,6 +118,7 @@ customizable. If your script won't change much, you can write your test
 incantation here directly:
 
 {% highlight yaml %}
+before_install: gem install jekyll html-proofer
```

Comment:
Shouldn't this just be the `install` step? My thought was that you'd add `html-proofer` to your Gemfile and Travis would run `bundle install` like usual.